rust-tdshim/src/asm/msr64.s

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2021 Intel Corporation
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+
+.section .text
+#  asm_read_msr64(
+#       index: u32, // rcx
+#       );
+.global asm_read_msr64
+asm_read_msr64:
+
+    rdmsr
+    shlq $0x20, %rdx 
```

Comment:
OOI, just wondering why some of the assembler files on this PR are in AT&T format whereas others are in Intel format?

Reply:
Yes, we had better be consistent.

Reply:
Fixed